docs/how-to-configure-new-repository.md

```diff
@@ -48,6 +48,9 @@ policies:
    repositores may opt-in for bigger number of required reviews.
 3. Every repo MUST enforce `cla/linuxfoundation` check for `Require status
    checks to pass before merging` section.
+   During the transition period, the CNCF `EasyCLA` check MAY be enabled.
```

Comment:
should the wording above "MUST enforce `cla/linuxfoundation` check" still remain?

Reply:
I can say `MAY be enabled instead of cla/linuxfoundation`, does that work?

Reply:
Since this is talking about configuring new repos, do we even need to mention the old CLA?

Reply:
No, because I think it also reflects our policy for existing repos, de facto.

Reply:
oh... the file is `docs/how-to-configure-new-repository.md`

in this case I would phrase it "at least one of these CLA checks MUST be required"

Reply:
PTAL